markymark/src/main/java/com/m2mobi/markymark/composer/DefaultMarkyMarkComposer.kt

```diff
@@ -366,7 +369,7 @@ open class DefaultMarkyMarkComposer : MarkyMarkComposer {
         ListItem(
             modifier = modifier.fillParentMaxWidth(),
             type = node.type,
-            children = node.children,
+            children = node.children.toImmutableList(),
```

Comment:
This is inside a `@Composable` and will run every recomposition, needs a remember

Reply:
Good catch. `node.children` already is an `ImmutableList` seeing as the Android Lint PR switches to the Kotlin Immutable Collections, I will fix it there